### ISSUE TITLE
Bashsupport-pro ask to set variable to local variable in a sh POSIX script

### ISSUE THREAD

**Issue by Etienne-Schmitt** [BashSupport-Pro/bashsupport-pro#78]
I've got a sh POSIX script like this:

```sh
#!/usr/bin/env sh

func() {
    echo "enter name"
    read -r name

    echo "$name"
}

func

```

But the highlighter keep telling me I could use $name as a local variable like this:
![image](https://user-images.githubusercontent.com/22135894/123814816-49fffe00-d8f6-11eb-9cb0-9f70aaf56a53.png)

But local is not (yet?) POSIX compliant.

Could you fix this ?

Thanks

**Comment by jansorg** [comment]
@Etienne-Schmitt Thanks for reporting! You're right that this shouldn't be shown for POSIX files.

This has already been fixed for the upcoming 2.0 version, https://plugins.jetbrains.com/plugin/13841-bashsupport-pro/next-version

I hope to publish 2.0 in about 4 weeks. Alternatively, you could use the latest beta of 2.0.

**Comment by jansorg** [comment]
2.0 has been released now.
